stellarphot/core.py

```diff
@@ -1089,7 +1089,7 @@ def _tidy_vizier_catalog(data, mag_column_regex, color_column_regex):
     @classmethod
     def from_vizier(
         cls,
-        header_or_center,
+        location,
```

Comment:
Given we use `location` elsewhere in the context of observatory, might I suggest 'field_center'?  Not critical, but seemed like a way to avoid confusion.

Reply:
Excellent suggestion, done!